app/views/pages/home.html.erb

```diff
@@ -1,71 +1,134 @@
+          <% @overview.last_org_files.each do |file| %>
+          <li class="my-3">
+            <%= bootstrap_icon(Settings.metadata_status[file.pod_metadata_status].icon_class, class: "pod-metadata-status #{file.pod_metadata_status} me-2") %>
+            <%= file.filename %>
           </li>
-          <li class="list-group-item border-0">
-            <%= link_to 'Uploading data using the API', 'https://github.com/pod4lib/aggregator/wiki/Uploading-data-using-the-API', class: 'text-muted' %>
+          <% end %>
+          <li class="my-3"><%= link_to t('.organization.uploads.all'), organization_stream_path(@overview.organization, @overview.organization.default_stream) %></li>
+        </ul>
+      </div>
+      <% end %>
+      <% if @overview.organization.provider? %>
+      <div class="my-2 my-md-0">
+        <h3>
+          <%= t('.organization.jobs.heading') %>
+        </h3>
+        <ul class="list-unstyled">
+          <% if @overview.active_org_jobs.present? %>
+          <% @overview.active_org_jobs.each do |status, jobs| %>
+          <li class="my-3">
+            <%= bootstrap_icon(Settings.job_status_group[status].icon_class, class: "pod-icon pod-job-tracker-status #{status} me-2")%>
+            <%= t('.organization.jobs.status_html', count: jobs.count, status: t("job_tracker.status_group.#{status}.label").downcase) %>
           </li>
-          <li class="list-group-item border-0">
-            <%= link_to 'More documentation (wiki)', 'https://github.com/pod4lib/aggregator/wiki', class: 'text-muted' %>
+          <% end %>
+          <% else %>
+          <li class="my-3">
+            <%= bootstrap_icon(Settings.job_status_group['completed'].icon_class, class: "pod-icon pod-job-tracker-status completed me-2")%>
+            <%= t('.organization.jobs.status_html.zero') %>
           </li>
+          <% end %>
+          <li class="my-3"><%= link_to t('.organization.jobs.all'), organization_path(@overview.organization) %></li>
         </ul>
       </div>
-    </div>
-    <div class="col pt-3 px-3 pt-md-5 px-md-5 text-center overflow-hidden">
-      <% if current_user %>
-        <div class="my-3 p-3">
-          <h2 class="display-5">My organizations</h2>
-          <ul class="list-group lead align-left">
-            <% organizations = Organization.with_roles([:member, :owner], current_user) %>
-
-            <% organizations.each do |org| %>
-              <li class="list-group-item border-0">
-                <%= link_to org.name, org, class: 'text-muted' %>
-              </li>
-            <% end %>
-          </ul>
-        </div>
       <% end %>
+      <div class="my-2 my-md-0">
+        <h3>
+          <%= t('.organization.actions.heading') %>
+        </h3>
+        <ul class="list-unstyled">
+          <% if @overview.organization.provider? %>
+          <li class="my-3"><%= link_to t('.organization.actions.upload_file'), new_organization_upload_path(@overview.organization) %></li>
```

Comment:
I think we need a permission check here because org members cannot currently upload files. I'm not 100% sure what behavior we want, but currently they cannot upload files.

Reply:
What's the right way to do that? Something like `if can? :create, Upload, { organization: @overview.organization, stream: @overview.organization.default_stream }`?

Reply:
I think this will work: `can? :create, @overview.organization.default_stream.uploads.build`

Reply:
I think that did it, thanks!